tests/scripts/test-ref-configs.pl

```diff
@@ -27,7 +27,8 @@
         'test_again_with_use_psa' => 1
     },
```

Comment:
I checked what `compat.sh` test cases we're running for each configuration here, and I think there are more omissions than the one I fixed, but for the other configurations we're running at least one test case so the CI is passing now:

* The CCM-PSK configurations aren't testing OpenSSL interoperability, but that's [a broader problem](https://github.com/Mbed-TLS/mbedtls/issues/9207).
* For suite-b, we aren't doing GnuTLS or OpenSSL interoperability. It's deliberate but I don't know why. [Both GnuTLS and OpenSSL interop fail, which I've filed as a separate issue](https://github.com/Mbed-TLS/mbedtls/issues/9206).